go.mod

```diff
@@ -63,3 +64,5 @@ require (
 )
 
 go 1.13
+
+replace github.com/pingcap/parser => github.com/imtbkcat/parser v0.0.0-20200918035743-76de33446dab
```

Comment:
?

Reply:
need merge  parser firstly, then i will delete the replace and update parser